practicals/large_language_models.ipynb

```diff
@@ -16,7 +16,7 @@
         "\n",
```

Comment:
<p>Did some changes in the text / formatting: </p><p> </p><pre class="ql-syntax" spellcheck="false"># @title **Paths to follow:** What is your level of experience in the topics presented in this notebook? (Run Cell)
experience = "advanced" #@param ["beginner", "intermediate", "advanced"]
# TODO: update guidance or remove this cell.


sections_to_follow=""


if experience == "beginner":
  sections_to_follow = (
  """we recommend you to **not** attempt to do every coding task but
instead, skip through to every section and ensure you interact with the
LoRA finetuned LLM presented in the last section as well as with the pretrained
LLM to get a practical understanding of how these models behave"""
  )
elif experience == "intermediate":
  sections_to_follow = (
  """we recommend you go through every section in this notebook
and try the coding tasks tagged as beginner or intermediate.
If you get stuck on the code ask a tutor for help or move on to better use the time of the practical"""
  )
elif experience == "advanced":
  sections_to_follow = (
      """we recommend you go through every section
and try every coding task until you get it to work""")


print(f"Based on your experience, {sections_to_follow}.\nNote: this is just a guideline, feel free to explore the colab as you'd like if you feel confortable!")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Changed